client/customer-effort-score-tracks/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import PropTypes from 'prop-types';
+import { recordEvent } from '@woocommerce/tracks';
+import CustomerEffortScore from '@woocommerce/customer-effort-score';
+
+/**
```

Comment:
Thanks for all the doc blocks and proptype declarations here. I also like this implementation to layer in tracks.